### ISSUE TITLE
Email signup displays variable as real name

### ISSUE THREAD

**Issue by wszarmach** [krico/jas#758]
Possibly we need to enforce the user to add a real name
![screen shot 2015-11-11 at 20 30 54](https://cloud.githubusercontent.com/assets/9398950/11100734/3c50d278-88b3-11e5-93e0-caab67c72674.png)

**Comment by krico** [comment]
When registering with email only there is no field to enter the real name...

The variable display fix is probably just a case of `$!{user.realName}` or `${!user.realName}`, don't remember the syntax but the `!` Means blank if null

**Comment by pauljanes** [comment]
Yep its raised in github already. The providers will always want a name. Its easier for them to check off who comes to class and its good customer service to know new peoples names when they turn up for the first time - Oh hello, you must be Waldemar.
